### ISSUE TITLE
Code and results for QuantumNAT missing

### ISSUE THREAD

**Issue by googlercolin** [mit-han-lab/torchquantum#22]
On the https://qmlsys.mit.edu page, the link to the source code for QuantumNAT directs to https://github.com/mit-han-lab/torchquantum.

I tried searching for the files in the repo manually, but I'm unable to find the .py.

**Comment by googlercolin** [comment]
Yes, I found QuantumNAS. However, I was referring to QuantumNAT (the one with the noisy gates added), not QuantumNAS. Is there your source code for this? @Hanrui-Wang

**Comment by Hanrui-Wang** [comment]
Hi Colin,

The `.py` files for QuantumNAS is available, for example, we can run `python examples/train.py examples/configs/mnist/four0123/train/super/u3cu3_s0/plain/blk8s1.1.1.yml` for the experiments.

For QuantumNAT, we are still working on a cleaner version of the code and will put that in the example folder. I will notify you as soon as it is ready (probably within 1~2 weeks). 

In the meanwhile, if you have any questions, we are also happy to answer. Thanks!
